src/vhdlConvertor/designFileParser.cpp

```diff
@@ -124,16 +125,37 @@ void VhdlDesignFileParser::visitContext_item(
 	// ;
 	auto l = ctx->library_clause();
 	if (l) {
-		NotImplementedLogger::print(
-				"DesignFileParser.visitContext_item - library_clause", l);
-		return; //libraries are ignored
+		visitLibrary_clause(l);
 	}
 	auto u = ctx->use_clause();
 	if (u) {
 		visitUse_clause(u, context.objs);
 	}
 }
 
+void VhdlDesignFileParser::visitLibrary_clause(
+		vhdlParser::Library_clauseContext *ctx) {
+	//rhinton:: I didn't create a method to visit a logical name 
+	// list and then an identifier list.  Since these are all part 
+	// of one statement without parsing options and not reused 
+	// anywhere, it seemed silly to create more functions to pass 
+	// through.
+	vhdlParser::Logical_name_listContext *lnl_ctx = ctx->logical_name_list();
+	if (lnl_ctx) {
```

Comment:
lnl_ctx is not optional, it can become null only if something is broken

Reply:
Agreed.  What if something is broken? :-)

Reply:
Something is broken == input VHDL is not valid -> Errors reported in exception, no other output from this file. So the convertor is actually not called if there is something broken and potentially some ANTLR AST elements are null.